components/Breadcrumb/src/breadcrumb.css

```diff
@@ -0,0 +1,125 @@
+  block-size: 100%;
+  display: flex;
+}
+
+ol.denhaag-breadcrumb__list {
+  list-style: none;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  padding-inline-start: 0;
+}
+
+.denhaag-breadcrumb__item,
+.denhaag-breadcrumb__link {
+  align-items: center;
+  display: flex;
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child) {
+  --denhaag-breadcrumb-background-color: var(--denhaag-breadcrumb-dots-background-color, var(--denhaag-color-grey-1));
+}
+
+.denhaag-breadcrumb__item:not(:first-child):not(:nth-last-child(2)):not(:last-child):hover {
+  --denhaag-breadcrumb-background-color: var(
+    --denhaag-breadcrumb-dots-hover-background-color,
+    var(--denhaag-color-grey-2)
+  );
+}
+
+.denhaag-breadcrumb__link {
+  color: var(--denhaag-breadcrumb-link-color, var(--denhaag-link-color));
+  padding-block-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-block-end: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-inline-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  padding-inline-end: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  pointer-events: none;
+  position: relative;
+}
+
+.denhaag-breadcrumb__link:hover,
+.denhaag-breadcrumb__link--hover {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-hover-color, var(--denhaag-link-hover-color));
+}
+
+.denhaag-breadcrumb__link:focus,
+.denhaag-breadcrumb__link--focus {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-link-focus-color, var(--denhaag-link-focus-color));
+  outline: var(--denhaag-breadcrumb-link-focus-outline, var(--denhaag-link-focus-outline));
+}
+
+.denhaag-breadcrumb__link--current,
+.denhaag-breadcrumb__link.denhaag-link--current,
+.denhaag-breadcrumb__item:last-child > .denhaag-breadcrumb__link {
+  --denhaag-breadcrumb-link-color: var(--denhaag-breadcrumb-current-color, var(--denhaag-breadcrumb-color));
+  text-decoration: none;
+}
+
+.denhaag-breadcrumb__link:not(.denhaag-breadcrumb__link--current):not(.denhaag-breadcrumb__link.denhaag-link--current):not(:last-child
+    > .denhaag-breadcrumb__link)
+  > :first-child {
+  pointer-events: bounding-box;
+}
+
+/* SVG for the separator */
+.denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  color: initial;
+  font-size: inherit;
+  margin-inline-start: var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs));
+  margin-inline-end: calc(-1 * var(--denhaag-breadcrumb-spacing, var(--denhaag-space-inline-xs)));
+}
+
+[dir="rtl"] .denhaag-breadcrumb__link > .denhaag-breadcrumb__text + .denhaag-icon {
+  transform: scaleX(-1);
+}
+
+.denhaag-breadcrumb__text {
+  background: var(--denhaag-breadcrumb-background-color);
+  overflow: hidden;
+  position: relative;
+  text-overflow: ellipsis;
+  transition: background var(--denhaag-breadcrumb-transition-ease) var(--denhaag-breadcrumb-transition-speed);
```

Comment:
Check, I am a fan of transitions since it makes an element just a little smoother behavior of the element since I find transitions quite hard. 

For now I'll remove the transition since it's not documented in Figma.